gnovm/pkg/doc/print.go

```diff
@@ -948,6 +800,16 @@ func (pkg *pkgPrinter) match(user, program string) bool {
 	return symbolMatch(user, program)
 }
 
+// Return the function signature, appended with " // crossing" if it's crossing.
+// However, don't append if pkg.opt.Source because the signature contains the body with crossing().
+func (pkg *pkgPrinter) crossingSignature(fun *JSONFunc) string {
+	if fun.Crossing && !pkg.opt.Source {
+		return fun.Signature + " // crossing"
+	} else {
+		return fun.Signature
+	}
+}
+
```

Comment:
Let's remove this; we can now understand if a function is crossing simply by looking at its parameters.